Header header
int32 num_detects
int32 closest
Detection[] detections

int32 xres
int32 yres
